src/components/app/list-new/AppList.tsx

```diff
@@ -40,9 +40,9 @@ export default function AppList() {
     const [showCreateNewAppSelectionModal, setShowCreateNewAppSelectionModal] = useState(false)
 
     // API master data
-    const [appCheckListRes, setAppCheckListRes] = useState({})
-    const [projectListRes, setProjectListRes] = useState({ result: [] })
-    const [environmentListRes, setEnvironmentListRes] = useState({ result: [] })
+    const [appCheckListRes, setAppCheckListRes] = useState({result: undefined});
```

Comment:
We should not set 'undefined' manually

Reply:
It's been replaced with `null` for now. However, we should be having some specific type for it.